### ISSUE TITLE
Plain theme panel no longer seems to work on 31.11.2

### ISSUE THREAD

**Issue by Airgetfrog** [KC3Kai/KC3Kai#2360]
A comment
=============
First off, I'll answer the format's questions just for show (hopefully so someone reads it sooner), but the issue seems pretty clear cut to me. Basically, the Plain 'theme' panel stopped working completely. I can only assume it's because of the changes implemented to support 7-ship fleet. 

Now, I understand that probably nobody even uses this version anymore, but that doesn't mean this isn't an issue (IMO there's a lot wrong with the new natsuiro theme, and I've posted about that to DJ a few months ago when he still worked on this, but I won't go into that here). I also get that you might find that it's not worth working on for a small minority of users, or that we should develop a solution ourselves - fair enough. We kept using the 'theme' with the compromise that it would be kept as legacy, and it finally broke. 

But that's where my current issue lies with this. Clearly, the theme is legacy. If no work is going to be done on it anymore, please actually label it as such, rather than 'Plain' (IIRC it was actually called legacy for a period of time). Also, from what I've seen looking at the code, it's not so much a theme as much as a different panel altogether. If so, please don't call it a theme. Let themes simply be a change in css if layout isn't supported, rather than also having different functionality. If at all possible, I think custom css would be quite nice, but that's just a wish.

tl;dr: Plain 'theme' broke, I understand not wanting to fix it, but at least call it Legacy and don't label it a theme.

### Versions:
Google Chrome | 62.0.3202.94 (Official Build) (64-bit) (cohort: Stable)
Revision | 4fd852a98d66564c88736c017b0a0b0478e885ad-refs/branch-heads/3202@{#789}
OS | Windows
(Win10, Version 1703)

KC3 version: Web store version 31.11.2/31.11.3 (Presumably .1 too, but I hadn't checked in a while)
Gameplay mode: Customize DMM Website

### Steps to reproduce:
1. Set Panel 'Theme' to Plain
2. Attempt to start the game via panel

### Actual behaviour:
Site doesn't load, panel doesnt load if site is started manually.

### Expected behaviour:
You understand.

### Screenshots (if applicable):
https://i.imgur.com/pXbXufY.png

**Comment by ethrundr** [comment]
I understand your point of view, but keep in mind that the one we'll update first is the "Natsuiro" theme, which is the currently widely used one.

I cannot promise for when we'll update the "Plain" with the new API, but at least I could say it will be done where there is enough time (and manpower) after we complete all the required adjustments and updates in "Natsuiro".

It was indeed used to be called as "Legacy", as at the time we shifted into "Natsuiro", it was decided not to update that theme anymore. However, as the time goes, several people requested for update on that theme, and we try to comply. Thus, the name changed back into "Plain". But, as per now, we are back to square one: lack manpower to keep both theme updated. We are still trying to keep both updated, but again, most of the time new updates and mechanism come before we can do anything to this "Plain" theme, which then will require us to update the "Natsuiro" first.

We'll see whether we will continue trying to keep this theme up, or just drop it altogether. Unfortunately, the decision is ultimately not on me, so I just can provide with explanations on why it is happened.
